### DISCUSSION TITLE
[@types/node] events.d.ts, options incomplete for events.on

### DISCUSSION THREAD

**Discussion by JohnnyBill** [DefinitelyTyped/DefinitelyTyped#69644] [Issues with a @types package]
The current state of the static events.on type definitions doesn't include the options, "close", "highWaterMark", "lowWaterMark".

See the documentation here: https://nodejs.org/api/events.html#eventsonemitter-eventname-options

Source code here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/76546c6bdb83437c68fa9e99267d29fcb7f37b91/types/node/events.d.ts#L261

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
Fixed in #69508
